### PR TITLE
Simplify outdate documentation

### LINKED ISSUE #285: 13Z055-90 HDLC: outdated documentation
README.TXT in 13Z055-90 is outdated. For example, it still references kernel 2.4 as only supported version.

Other things to add/check:
 - How to write address and control bytes? (-> Just prepend them to actual payload and pass to tty device)
 - Maybe refer reader to z055_hdlc_util's usage print for configuration options.
 - Besides the outdated kernel version, is the rest of the README.TXT (also README.PPP!) still valid?
 
 MAIN_PR008429

### PR DESCRIPTION
**Pull request by mad-jsanjuan**
Remove references to old methods for compiling these components agains older linux kernels and ELINOS projects.

Keep useful stuff but remove things that may lead to confusing documentation and may get outdated by third parties (installing other tools).

### CODE REVIEW

**Review by mad-jrodriguez — APPROVED** [review]

File: DRIVERS/Z055_HDLC/README.md [hunk 1]
```diff
@@ -27,6 +26,11 @@ README.PPP            Setting up a PPP network connection (using pppd program)
 *** Applications ***
 ********************
 
+This driver is intended for use in synchronous HDLC mode.
+All packages written to the tty device that this z055_hdlc_drv
+driver creates should contain an entire HDLC frame. Including
```
Comment (ckloecker) [inline DRIVERS/Z055_HDLC/README.md]: No, not "an entire HDLC frame" as the driver takes care of the flags and checksum. Rather:

The z055_hdlc_drv driver takes the HDLC address and control bytes from the beginning of each data package that is written to the tty device. So all packages written to the tty device should contain those bytes. For example, "echo -ne '\x13\x37Hello world!' > /dev/ttyTH0" will embed "Hello world!" in an HDLC package with address 0x13 and control 0x37.

File: DRIVERS/Z055_HDLC/README.md [hunk 2]
```diff
@@ -74,74 +78,8 @@ Public License (GPL).
...
-The programs can be built in the Z055_HDLC
-distribution directory by performing the following steps
-
-In the $(ELINOS_PROJECT)/src/mdis directory add the driver.mak file
-to the key ALL_NATIVE_TOOLS
-(e.g. ALL_NATIVE_TOOLS = DRIVERS/Z055_HDLC/Z055_HDLC_UTIL/program.mak)
-
-then build / compile the project from the $(ELINOS_PROJECT) directory type
-# make boot
+These Z055 HDLC drivers and tools shall be built inside a MDIS
```
Comment (ckloecker) [inline DRIVERS/Z055_HDLC/README.md]:
```suggestion
These Z055 HDLC drivers and tools shall be built inside an MDIS
```

File: DRIVERS/Z055_HDLC/README.md [hunk 3]
```diff
@@ -74,74 +78,8 @@ Public License (GPL).
...
+Project from the parent component 13MD05-90.
```
Comment (ckloecker) [inline DRIVERS/Z055_HDLC/README.md]:
```suggestion
project from the parent component 13MD05-90.
```

File: DRIVERS/Z055_HDLC/README.md
Comment (ckloecker) [inline DRIVERS/Z055_HDLC/README.md:1]: FIXME: README.md still references README.PPP that has been renamed to PPP.md!

File: DRIVERS/Z055_HDLC/PPP.md [hunk 4]
```diff
@@ -103,12 +63,9 @@ deactivates the connection.
 
 
 --------------------------
-PPP for Linux Kernel 2.4.x
+PPP for Linux Kernel
 --------------------------
```
Comment (ckloecker) [inline DRIVERS/Z055_HDLC/PPP.md]:
```suggestion
--------------------
```

File: DRIVERS/Z055_HDLC/PPP.md [hunk 5]
```diff
@@ -103,12 +63,9 @@ deactivates the connection.
 
 
 --------------------------
```
Comment (ckloecker) [inline DRIVERS/Z055_HDLC/PPP.md]:
```suggestion
--------------------
```

**Review by ckloecker — CHANGES_REQUESTED** [review]

**Comment by mad-jsanjuan** [comment]
@ckloecker Thanks for the review. I have addressed the issues you pointed out.

File: DRIVERS/Z055_HDLC/README.md [hunk 6]
```diff
@@ -20,13 +19,20 @@ z055_hdlc_util Configuration Utility
...
 
 
 ********************
 *** Applications ***
 ********************
 
+This driver is intended for use in synchronous HDLC mode.
+All packages written to the tty device that this z055_hdlc_drv
+driver creates should contain the address and control bytes at
+the beggining of the frame and then the data payload. The checksum
```
Comment (ckloecker) [inline DRIVERS/Z055_HDLC/README.md]:
```suggestion
the beginning of the frame and then the data payload. The checksum
```

**Pull request merged**